ansible/group_vars/all.yml

```diff
@@ -204,20 +151,16 @@ elao_php_fpm_pools:
     template: fpm_pools/www_{{ _env }}.conf.j2
 
 elao_php_extensions:
-  # Dev
-  - xdebug
```

Comment:
The extension is not upstream ready for php7. I will add it back again when it's ready.